docs/api.md

```diff
@@ -357,7 +357,8 @@ Adjust various settings related to the app display.
 
 ### JSON Properties
 
-You can adjust each property in the JSON object according to your preferences. Including a property is optional.
+You can adjust each property in the JSON object according to your preferences. Including a property is optional. 
+✔ Please note, that all keynames are case sensitiv.
```

Comment:
IMHO it should `key names` and `sensitiv_e_`

Reply:
You're right. Thank you.